http-api/src/lib.rs

```diff
@@ -389,22 +398,53 @@ async fn remote_handler(
 async fn history_handler(
     ctx: Context,
     project: Urn,
-    qs: HashMap<String, String>,
+    qs: CommitsQueryString,
 ) -> Result<impl Reply, Rejection> {
-    let (sha, fallback_to_head) = match qs.get("from").cloned() {
+    let CommitsQueryString {
+        since,
+        until,
+        parent,
+    } = qs;
+
+    let (sha, fallback_to_head) = match parent {
         Some(commit) => (commit, false),
         None => {
             let meta = project_info(project.to_owned(), ctx.paths.to_owned())?;
             (meta.head.to_string(), true)
         }
     };
+
     let reference = Reference::head(
         Namespace::from(project),
         None,
         One::from_str(&sha).map_err(|_| Error::NotFound)?,
     );
+
     let commits = browse(reference, ctx.paths, |browser| {
-        radicle_source::commits::<PeerId>(browser, None)
+        let mut result = radicle_source::commits::<PeerId>(browser, None)?;
+        let headers: Vec<Header> = result
+            .headers
+            .iter()
+            .filter(|q| {
+                if let (Some(since), Some(until)) = (since, until) {
+                    q.committer_time.seconds() > since && q.committer_time.seconds() < until
+                } else if let Some(since) = since {
+                    q.committer_time.seconds() >= since
+                } else if let Some(until) = until {
+                    q.committer_time.seconds() < until
+                } else {
+                    // If the commitTime is neither after `since` nor before `until` we include the commit header
```

Comment:
Shouldn't this comment say "If neither `since` nor `until` are specified, we include the commi"?